openmetadata-service/src/main/java/org/openmetadata/service/search/indexes/DashboardIndex.java

```diff
@@ -76,6 +65,17 @@ public Map<String, Object> buildESDoc() {
     doc.put("entityType", Entity.DASHBOARD);
     doc.put("serviceType", dashboard.getServiceType());
     doc.put("fqnParts", suggest.stream().map(SearchSuggest::getInput).collect(Collectors.toList()));
+    if (dashboard.getOwner() != null) {
```

Comment:
@07Himank why do we need to do this? we are just capturing the owner entity reference as it is today